android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java

```diff
@@ -481,4 +482,11 @@ public void createNewSession(String trackerId, String screenName) {
         tracker.setScreenName(screenName);
         tracker.send(new HitBuilders.ScreenViewBuilder().setNewSession().build());
     }
+
+    @ReactMethod
+    public void dispatch(Promise promise) {
+        GoogleAnalytics analytics = GoogleAnalytics.getInstance(getReactApplicationContext());
+        analytics.dispatchLocalHits();
```

Comment:
Please wrap this call in a try/catch in case it fails, and do a promise.reject if it does.

E.g.
```java
try {
    analytics.dispatchLocalHits();
    promise.resolve(true);
} catch (Exception ex) {
    promise.reject(ex);
}
```